examples/all-clusters-app/nrfconnect/main/main.cpp

```diff
@@ -33,9 +33,9 @@ static int InitUSB()
 {
     int err = usb_enable(nullptr);
 
-    if (err)
+    if ((err != 0) && (err != -EALREADY))
```

Comment:
ok, so this failed because `USB_DEVICE_INITIALIZE_AT_BOOT` had been enabled for the dongle by default. We could probably also remove the manual init completely, but your fix is fine as well :)